src/sxe/sxeedit.cpp

```diff
@@ -18,6 +19,7 @@
  */
 
 #include "sxeedit.h"
+
```

Comment:
empty line? for what?

Reply:
Yes to use the same logic in all places, already in place to several files.
And to seperate the .h #include of the same file, there is no <include> in this file.